src/pages/patientView/copyNumberAlterations/CopyNumberTableWrapper.tsx

```diff
@@ -33,6 +49,15 @@ export default class CopyNumberTableWrapper extends React.Component<{ store:Pati
         });
 
         columns.push({
+            name: "CNA",
+            render: (d:DiscreteCopyNumberData)=><span>{renderAlterationTypes(d.alteration)}</span>,
+            download: (d:DiscreteCopyNumberData)=>(AlterationTypes[d.alteration]),
+            sort: (d1:DiscreteCopyNumberData, d2:DiscreteCopyNumberData, ascending:boolean)=>0,
```

Comment:
We will need to revisit this (and all other column definitions in this table) after switching to the new version of `MSKTable`